src/blog/components/informationMessage/InformationMessage.tsx

```diff
@@ -0,0 +1,67 @@
+import Button from "src/components/buttons/Button";
+import LinkButton from "src/components/linkButton/LinkButton";
+import Text from "src/components/text/Text";
+import { ILink, LinkType } from "studio/lib/payloads/navigation";
+import styles from "./informationMessage.module.css";
+import React from "react";
+
+export const homeLink = {
+  _key: "return-home",
+  _type: "link",
+  linkTitle: "Return to home",
+  linkType: LinkType.Internal,
+  internalLink: {
+    _ref: "/",
+  },
+};
+
+export const studioLink = {
+  _key: "go-to-studio",
+  _type: "link",
+  linkTitle: "Go to studio",
+  linkType: LinkType.Internal,
+  internalLink: { _ref: "/studio" },
+};
+
+interface InformationMessageProps {
+  title: string;
+  body: string;
+  button?: {
+    label: string;
+    onClick: () => void;
+  };
+  link?: ILink;
+}
+
+const InformationMessage = ({
+  title,
+  body,
+  button,
+  link,
+}: InformationMessageProps) => {
+  return (
+    <section className={styles.wrapper} role="status" aria-live="assertive">
```

Comment:
aria isn't really needed here as it's not a dialogue or an important message, it's more general information, like any section.. though we might want to rename from Information**Message** to something less rigid, to... InformationBox -View, -Screen 🤷‍♀️  not sure I like anyof those 😅 

```suggestion
    <section className={styles.wrapper}>
```

Reply:
I renamed it to informationSection. Don't agree with myself it thats any better than infomationBox or view etc.